README.md

```diff
@@ -16,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://bonkegcobo.github.io/LeaderBoard/)
```

Comment:
- [ ] I would suggest deploying the last version of your app in order to maintain your README and repository up-to-date because it could be confusing for users if they open the unfinished version. Please, kindly deploy the last version of your app :wink:
- [ ] I think it could be a good idea to update your README with the last information of your project. Remember that this is the first thing recruiters/users will see when checking your project and it should include all the information related to your project. Please, kindly update your README information including the screenshots and API instructions that your app consumes :smiley: :raised_hands: